proxy/envoy/config.go

```diff
@@ -184,13 +184,28 @@ func buildOutboundFilters(instances []*model.ServiceInstance, services []*model.
 	suffix := sharedInstanceHost(instances)
 	httpConfigs := make(HTTPRouteConfigs)
 
+	// get all the route rules applicable to the instances
+	rules := config.RouteRulesBySource("", instances)
```

Comment:
That's true. We really need to rename these variables in future for better readability. May be call these as localSvcInstances ?